docs/sources/panels-visualizations/query-transform-data/transform-data/index.md

```diff
@@ -39,6 +39,7 @@ labels:
     - enterprise
     - oss
 title: Transform data
+description: Use transformations to manipulate your data before visualizations are applied
```

Comment:
maybe?
```suggestion
description: Use transformations to manipulate your data before being applied in visualizations
```

Reply:
Not sure. I based it on the first few sentences of the body text. Grafana applies the transformation to the data before it applies the visualization. That's what this is meant to say. But we can also just take that part out entirely.
```suggestion
description: Use transformations to manipulate your data
```

Reply:
Another option
```suggestion
description: Use transformations to rename fields, join series data, apply mathematical operations, and more
```